frontend/src/components/interview/components/other-applications/helpers/helpers.ts

```diff
@@ -0,0 +1,2 @@
+export { getOtherApplicationsColumns } from './column-helpers/column-helpers';
+export { getOtherApplicationsRows } from './row-helpers/row-helpers';
```

Comment:
```suggestion
export { getOtherApplicationsColumns } from './get-other-applications-rows/get-other-applications-rows.helper.ts';
export { getOtherApplicationsRows } from // same
```
each helper should be in the separate folder. we don't want separate folders like `column-helpers`